node/src/effect.rs

```diff
@@ -695,6 +696,22 @@ impl<REv> EffectBuilder<REv> {
             .await
     }
 
+    /// Announce a committed Step result
+    pub(crate) async fn announce_step_result(self, era_id: EraId, execution_effect: ExecutionEffect)
```

Comment:
It's not so much a step result as a step success:
```suggestion
    /// Announce a Step success.
    pub(crate) async fn announce_step_success(self, era_id: EraId, execution_effect: ExecutionEffect)
```

Reply:
Applied suggestion